client/tests/RequestTerm.integ.test.js

```diff
@@ -16,9 +13,7 @@ global.fetch = mockResponse();
 
 it('Renders form and submit term', async () => {
   // Arrange
-  const defaultQueries = customQueries.default
-  const { getByText, getByTestId } = render(<RequestForm />,
-    { queries: { ...queries, ...defaultQueries }, })
```

Comment:
what are these for? the test doesn't seem to care if you delete them

Reply:
So, I created a customized query for getByText to retrieve a simple error message that I was able to compare with when trying to find the Text inside the snackbar, but the snackbar is not there anymore; so we are expecting an error. The getByText query from react-integ-test returns a complex error message containing the HTML of the whole form, which is mostly a debug message, but here we care about throwing the error and assert the message.